ui/components/multichain/multi-srp/select-srp/index.scss

```diff
@@ -0,0 +1,5 @@
+@use "design-system";
+
+.select-srp__container {
+  cursor: pointer;
```

Comment:
@georgewrmarshall Would it make sense to add a prop to `Box` to accommodate cursors?

Reply:
If we implement TailwindCSS in the extension, we won’t have to update the `Box` component or create SCSS stylesheets ever again! 🎉 [Slack post](https://consensys.slack.com/archives/C1L7H42BT/p1741376714071459)